.eslintrc.json

```diff
@@ -0,0 +1,33 @@
+{
+    "root": true,
+    "extends": "airbnb-base",
+    "env": {
+      "node": true,
+      "es6": true,
+      "mocha": true
+    },
+    "rules": {
+      "one-var": 0,
+      "one-var-declaration-per-line": 0,
+      "new-cap": 0,
+      "consistent-return": 0,
+      "no-param-reassign": 0,
+      "comma-dangle": 0,
+      "curly": ["error", "multi-line"],
+      "import/no-unresolved": [2, { "commonjs": true }],
+      "no-shadow": ["error", { "allow": ["req", "res", "err"] }],
+      "valid-jsdoc": ["error", {
+        "requireReturn": true,
+        "requireReturnType": true,
+        "requireParamDescription": false,
```

Comment:
Any reason why `requireParamDescription` is false?

Reply:
Oh Sorry! meant for it to be true